### ISSUE TITLE
Bot instantly deletes webhook message

### ISSUE THREAD

**Issue by Raplhs** [Smug246/Luna-Token-Grabber#17]
I built my exe, but when I run it, the bot sends a message through webhook, but then instantly deletes it.

**Comment by GodFather227** [comment]
> Raplhs

can you take a vid of it or something

**Comment by Smug246** [comment]
> I built my exe, but when I run it, the bot sends a message through webhook, but then instantly deletes it.


are you rate limited or do you have any bot in that server that could be deleting

**Comment by Raplhs** [comment]
Sorry again, I was up at around 4am trying to fix my other issue, but caused this error. Nothing is wrong, just me in the brain last night lol.

**Comment by Raplhs** [comment]
I closed the thread because I thought I found a solution, however I ran into it again. It turns out it's not because I removed something from the code, I think it's because when someone downloads it, they'll run it in the same folder as the download. In this case I put it in downloads as if I just downloaded it, but then the bot sends the message, then deletes the message, then deletes the webhook. (I forgot to show the webhook being deleted, but it does get deleted, just doesn't show in audit logs.)

https://user-images.githubusercontent.com/103277643/183252643-20a9849a-8933-41b8-81d1-8e973dd046f8.mp4
